classes/Gender.php

```diff
@@ -31,6 +31,10 @@
  */
 class GenderCore extends ObjectModel
 {
+    public const GENDER_MALE = '0';
+    public const GENDER_FEMALE = '1';
+    public const GENDER_NEUTRAL = '2';
```

Comment:
Suggestion:

```suggestion
    public const GENDER_TYPE_MALE = '0';
    public const GENDER_TYPE_FEMALE = '1';
    public const GENDER_TYPE_NEUTRAL = '2';
```

To emphasize the fact that it's linked to the type field

Reply:
Fixed